docs/en_us/platform_api/source/mobile/overview.rst

```diff
@@ -0,0 +1,44 @@
+.. _edX Platform Mobile API Overview:
+
+################################################
+EdX Platform Mobile API Overview
+################################################
+
+Use the edX Platform Mobile API to build mobile applications for students to
+view course information and videos for courses on your instance of Open edX.
+
+The edX Platform Mobile API uses Representational State Transfer (REST) design
+principles and supports JavaScript Object Notation (JSON) data-interchange
+format. The REST API is simple, lightweight, and optimized.
+
+******************************************
+EdX Platform Mobile API Version 0.5, Alpha
+******************************************
+
+The edX Platform Mobile API is currently at version 0.5 and is an Alpha
```

Comment:
I like this description for the state of the API. Could you use essentially the same thing for the User API too.